src/jit/hwintrinsicxarch.cpp

```diff
@@ -781,9 +781,59 @@ GenTree* Compiler::impSSE2Intrinsic(NamedIntrinsic        intrinsic,
     GenTree*  retNode  = nullptr;
     GenTree*  op1      = nullptr;
     GenTree*  op2      = nullptr;
+    int       ival     = -1;
+    int       simdSize = simdSizeOfHWIntrinsic(intrinsic, sig);
     var_types baseType = TYP_UNKNOWN;
+    var_types retType  = TYP_UNKNOWN;
+
     switch (intrinsic)
     {
+        case NI_SSE2_CompareLessThan:
+            assert(sig->numArgs == 2);
+            op2      = impSIMDPopStack(TYP_SIMD16);
+            op1      = impSIMDPopStack(TYP_SIMD16);
+            baseType = getBaseTypeOfSIMDType(sig->retTypeSigClass);
+            switch (baseType)
+            {
+                case TYP_DOUBLE:
+                    retNode = gtNewSimdHWIntrinsicNode(TYP_SIMD16, op1, op2, intrinsic, baseType, simdSize);
+                    break;
+                case TYP_BYTE:
+                case TYP_SHORT:
+                case TYP_INT:
+                    retNode = gtNewSimdHWIntrinsicNode(TYP_SIMD16, op2, op1, intrinsic, baseType, simdSize);
```

Comment:
As I said previous, here we should generate a node of `gtNewSimdHWIntrinsicNode(TYP_SIMD16, op2, op1, SSE2_CompareGreaterThan, baseType, simdSize);`

Reply:
Can you point me to that comment?

Reply:
https://github.com/dotnet/coreclr/pull/15585#discussion_r163116269

Reply:
Looks like I have overlooked this one - will fix